service/gcs/prot/protocol.go

```diff
@@ -414,6 +421,15 @@ type MappedVirtualDisk struct {
 	ReadOnly          bool  `json:",omitempty"`
 }
 
+// MappedDirectory represents a directory on the host which is mapped to a
+// directory on the guest through a technology such as Plan9.
+type MappedDirectory struct {
+	ContainerPath     string
+	CreateInUtilityVM bool   `json:",omitempty"`
```

Comment:
What is the use of this? Given that we only allow you to pass it as `false` today in `storage.go`